easybuild/tools/robot.py

```diff
@@ -379,13 +379,24 @@ def resolve_dependencies(easyconfigs, modtool, retain_all_deps=False):
     return ordered_ecs
 
 
-def search_easyconfigs(query, short=False, filename_only=False, terse=False):
-    """Search for easyconfigs, if a query is provided."""
+def search_easyconfigs(query, short=False, filename_only=False, terse=False, return_robot_list=False):
```

Comment:
@ocaisa Wouldn't two separate arguments make more sense, for better control?

* `consider_extra_paths=True` to control whether or not to include the paths listed in `--search-paths`
* `return_list=False` to control whether or not a list is returned rather than being printed
   (or maybe `print_result=True` is better, and always return the list regardless of whether or not printing is being done)